recipes/recipes_emscripten/highfive/build.sh

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+# Configure step
+emcmake cmake ${CMAKE_ARGS} ..      \
+    -DCMAKE_POLICY_VERSION_MINIMUM=3.5 \
+    -DCMAKE_PREFIX_PATH:PATH=${PREFIX} \
+    -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DHIGHFIVE_UNIT_TESTS=OFF \
+    -DHIGHFIVE_EXAMPLES=OFF \
+    -DHIGHFIVE_USE_BOOST=OFF \
+    -DHIGHFIVE_USE_INSTALL_DEPS=OFF \
+    -DCMAKE_PROJECT_INCLUDE=$RECIPE_DIR/overwriteProp.cmake
+
+# Build step
+make install
```

Comment:
Prefer emmake make and install over make install here.

Reply:
Ok I add `emmake make` and `emmake make install`